winch/codegen/src/trampoline.rs

```diff
@@ -230,7 +231,9 @@ where
         if wasm_sig.has_stack_results() {
             self.masm.free_stack(wasm_sig.results.size());
         }
-        self.epilogue_with_callee_saved_restore(spill_size);
+
+        self.masm.free_stack(spill_size);
+        self.masm.epilogue(&self.callee_saved_regs, 0);
```

Comment:
Could we do `self.masm.epilogue(&self.callee_saved_regs, spill_size)` here?

Reply:
We could, but at that point i wonder if we should move the stack allocation out of  the prologue and epilogue. What do you think about not including the reserve/free calls?

Reply:
Yeah, that makes sense to me as well.